lib/peeps.rb

```diff
@@ -0,0 +1,39 @@
+require 'pg'
+# require 'Time'
+
+class Peeps
+
+  attr_reader :id, :peep, :time, :user_id, :user_name
+
+  def initialize(id:, peep:, time:, user_id:, user_name: User)
+    @id = id
+    @peep = peep
+    @time = time
+    @user_id = user_id
+    # @user_name = user_name.find(id: user_id).username
+  end
+
+  def self.create(peep:, user_id:)
+    return nil unless peep
+
+    if ENV['Environment'] == 'test'
+      connection = PG.connect(dbname: 'chitter_chatter_test')
+    else
+      connection = PG.connect(dbname: 'chitter_chatter')
+    end
+    result = connection.exec("INSERT INTO peeps (peep) VALUES('#{peep}') RETURNING id, peep, time, user_id;").first
+    Peeps.new(id: result["id"], peep: result["peep"], time: Time.parse(result["time"]), user_id: result['user_id'])
```

Comment:
I don't understand how you've done the time but it looks smart! Its nice and need on the web page as well